specifyweb/frontend/js_src/lib/hooks/useCollection.tsx

```diff
@@ -71,7 +75,10 @@ export function useCollection<SCHEMA extends AnySchema>({
               setCollection(collection);
           },
         } as CollectionFetchFilters<AnySchema>)
-        .catch(raise);
+        .catch((error: Error, ...args: RA<unknown>) => {
```

Comment:
.catch() callback is only ever called with one argument

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch#reason

so the others here are needless

or is this some sort of special backbone jquery promise?

Reply:
Ah: no nothing special happening here. 
I was initially getting a TypeScript error when calling `raise` without at least one other argument. 
When rewriting the code here, I'm no longer getting the error so must have been something with the compiler?